aadhaar_qr_verifier/src/tests/mod.nr

```diff
@@ -0,0 +1,171 @@
+            205, 22, 186, 92, 163, 108, 70, 25, 168, 130, 96, 162, 13, 106, 78, 71, 254, 89, 108,
+            233, 225, 39, 97, 63, 17, 13, 42, 95, 0, 30, 0, 130, 28, 154, 5, 162, 1, 51, 178, 125,
+            65, 235, 47, 162, 24, 112, 152, 119, 13, 83, 212, 102, 84, 248, 36, 193, 63, 193, 101,
+            254, 213, 80, 59, 106, 79, 74, 170, 227, 110, 2, 204, 201, 182, 60, 114, 60, 65, 195,
+            148, 232, 145, 30, 46, 232, 19, 28, 159, 29, 153, 101, 206, 162, 28, 108, 80, 229, 255,
+            145, 0, 4, 0, 13, 128, 255, 145, 0, 4, 0, 14, 128, 255, 145, 0, 4, 0, 15, 195, 224, 188,
+            135, 193, 127, 15, 134, 196, 158, 62, 204, 245, 173, 53, 224, 68, 102, 215, 173, 58,
+            244, 81, 11, 131, 110, 245, 166, 156, 166, 215, 76, 158, 156, 10, 170, 148, 182, 60,
+            115, 173, 176, 16, 1, 15, 169, 187, 111, 38, 226, 185, 241, 97, 88, 34, 140, 218, 123,
+            203, 88, 112, 88, 143, 74, 108, 17, 106, 37, 69, 149, 69, 151, 215, 37, 209, 201, 125,
+            81, 117, 124, 102, 108, 252, 251, 26, 14, 104, 229, 35, 22, 219, 199, 180, 110, 24, 47,
+            126, 104, 141, 232, 237, 115, 47, 31, 176, 190, 111, 243, 116, 185, 32, 185, 121, 57,
+            59, 197, 65, 61, 233, 98, 18, 172, 235, 82, 146, 150, 241, 30, 186, 172, 161, 155, 148,
+            171, 212, 237, 170, 123, 85, 177, 76, 132, 230, 38, 114, 51, 254, 36, 37, 236, 202, 21,
+            44, 48, 203, 191, 154, 22, 219, 13, 67, 28, 214, 71, 72, 63, 173, 24, 137, 62, 141, 14,
+            123, 11, 237, 106, 219, 68, 40, 62, 29, 255, 145, 0, 4, 0, 16, 128, 255, 145, 0, 4, 0,
+            17, 128, 255, 217, 235, 89, 152, 53, 59, 33, 186, 193, 56, 29, 117, 14, 31, 162, 102,
+            65, 224, 150, 175, 57, 29, 17, 95, 173, 173, 212, 123, 244, 250, 47, 255, 139, 96, 54,
+            34, 219, 243, 92, 100, 240, 182, 255, 21, 167, 91, 244, 127, 53, 165, 180, 223, 131,
+            221, 184, 74, 53, 233, 255, 238, 30, 166, 5, 157, 243, 128, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+            0, 0, 0, 35, 136,
+        ];
+
+        let mut msg_vector: BoundedVec<u8, MAX_DATA_LENGTH> = BoundedVec::new();
+
+        for i in 0..signed_qr_data.len() {
+            msg_vector.push(signed_qr_data[i]);
+        }
+
+        println(msg_vector.storage);
+
+        let delimiter =
+            [2, 4, 26, 38, 49, 51, 68, 79, 80, 96, 97, 104, 118, 124, 152, 165, 179, 184];
+        let nullifierSeed: Field = 12345678; // Example seed
+        let signalHash = 1001;
+
+        let outputs = main(
+            msg_vector,
+            signed_data_length,
+            delimiter,
+            signature_limbs,
+            modulus_limbs,
+            redc_limbs,
+            true,
+            true,
+            true,
+            true,
+            nullifierSeed,
+            signalHash,
+        );
+
+        // all of these result values are extracted from circom codebase.
+        assert(outputs.age, "Age should be above 18");
```

Comment:
There is no assert happening here
I guess you wanted to write 
`assert(outputs.age == true, "Age should be above 18");`

Reply:
yeah fixed it